ontology-engine/graph-core_2.11/src/main/scala/org/sunbird/graph/util/CSPMetaUtil.scala

```diff
@@ -0,0 +1,74 @@
+package org.sunbird.graph.util
+
+import org.apache.commons.collections4.MapUtils
+import org.slf4j.LoggerFactory
+import org.sunbird.common.Platform
+import org.sunbird.common.dto.Property
+import org.sunbird.graph.dac.model.Node
+
+import scala.collection.JavaConverters._
+
+object CSPMetaUtil {
+  private[this] val logger = LoggerFactory.getLogger(classOf[CSPMetaUtil])
+  private val defaultMetadataList = java.util.Arrays.asList("appIcon","posterImage","artifactUrl","downloadUrl","variants","previewUrl","pdfUrl")
+  private val defaultWriteBasePath = java.util.Arrays.asList("https://sunbirddev.blob.core.windows.net","https://obj.dev.sunbird.org")
+
+  def updateAbsolutePath(data: java.util.Map[String, AnyRef]): java.util.Map[String, AnyRef] = {
+    val cspMeta = Platform.getStringList("cloudstorage.metadata.list", defaultMetadataList).asScala.toList
+    val absolutePath = Platform.getString("cloudstorage.read_base_path", "") + java.io.File.separator + Platform.getString("cloud_storage_container", "")
+    if (MapUtils.isNotEmpty(data)) {
+      val updatedMeta: java.util.Map[String, AnyRef] = new java.util.HashMap[String, AnyRef]
+      data.asScala.map(x =>
+        if (cspMeta.contains(x._1))
+          updatedMeta.put(x._1, x._2.asInstanceOf[String].replace("CLOUD_STORAGE_BASE_PATH", absolutePath))
```

Comment:
@Jayaprakash8887 Make a note as part of CSP migration we should replace the base storage path/host as 
CLOUD_STORAGE_BASE_PATH